src/Common/src/System/Security/Cryptography/Asn1/AlgorithmIdentifierAsn.xml.cs

```diff
@@ -1,4 +1,8 @@
-﻿using System;
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@bartonjs, were these .xml.cs files generated by a tool?  Assuming yes, is that tool checked in somewhere?

Reply:
Yes, https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Security/Cryptography/Asn1/asn.xslt

Reply:
Thanks